packages/core/helper-plugin/src/components/Status.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styled from 'styled-components';
+
+type BulletProps = {
+  backgroundColor: 'alternative600' | 'danger600' | 'neutral600' | 'primary600' | 'secondary600' | 'success600' | 'warning600';
+}
+
+const Bullet = styled.div<BulletProps>`
+  margin-right: ${({ theme }) => theme.spaces[3]};
+  width: ${6 / 16}rem;
+  height: ${6 / 16}rem;
+  border-radius: 50%;
+  background: ${({ theme, backgroundColor }) => theme.colors[backgroundColor]};
+`;
+
+type StatusProps = {
+  variant: 'alternative' | 'danger' | 'neutral' | 'primary' | 'secondary' | 'success' | 'warning';
```

Comment:
@todo Refer to a shared/common `variant` type

Reply:
you can check [Status](https://github.com/strapi/design-system/blob/main/packages/strapi-design-system/src/Status/Status.tsx) component from DS which exports these props. 
@markkaylor Could you please confirm if this component needs to deprecated or not as we have same component ( may be with +/- functionalities) in DS.

Reply:
Good catch. Yeah I think we can @deprecated with a message to use the Design System component instead. 

There is still [one spot](https://github.com/strapi/strapi/blob/main/packages/core/admin/admin/src/pages/SettingsPage/pages/Users/ListPage/utils/tableHeaders.js) in the Strapi codebase that is using an `isActive` prop that is not even supported by the helper-plugin Component as far as I can tell. I guess we can update to use the DS one.

Reply:
Ready for feedback